cluster/cluster.yaml

```diff
@@ -3222,6 +3222,217 @@ Resources:
           PolicyName: root
       RoleName: "{{.Cluster.LocalID}}-audittrail-adapter"
     Type: 'AWS::IAM::Role'
+# {{- if eq .Cluster.Provider "zalando-eks"}}
```

Comment:
Should we also condition all this to `eks_ip_family: "ipv6"`? Since on IPv4 the normal "in-tree" controller works.

Reply:
I didn't think about that case. We could do it, but it should also work like it is now. `admission-controller` shifts _all_ Services with LoadBalancer type to the new controller on EKS, regardless of IP version.

So, it depends on which controller we prefer. I think there's benefit in using a single controller for EKS rather than two based on IP version.

Do we intend to run IPv4 EKS clusters going forward?

Reply:
Given that we have existing EKS/IPv4 clusters it's probably better to add the check.